### ISSUE TITLE
[Feature] Add additional styling to the contributors page

### ISSUE THREAD

**Issue by karthiknadar1204** [rupali-codes/LinksHub#1236]
### Description

The current contributors page look beautiful, I though of further styling it by adding some hover effects to the cards and the buttons to the cards to make them look more engaging.

Please feel share your opinion or any other styling you wish for me to apply

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

**Comment by rupali-codes** [comment]
@karthiknadar1204 we appreciate your willingness to contribute though, but don't start working before you get assigned to any issue :)

**Comment by karthiknadar1204** [comment]
Yes,done with it though,please review my pr and once done,please assign this issue to me.

Thank you

**Comment by karthiknadar1204** [comment]
@CBID2 Can I be assigned to this issue now, The pr is almost wrapped up with one review left, Want to continue contributing to the project.

**Comment by karthiknadar1204** [comment]
@rupali-codes @CBID2 I'll open a new pr for the same for this issue discarding the previous additional files and adding the new proposed changes.
